### PR TITLE
update border radius control design

### PR DESCRIPTION
**Pull request by bkrmendy**
Update border radius control design along [this CSB](https://codesandbox.io/s/canvas-control-templates-s3i69?file=/src/App.js:1620-1651)

image

### CODE REVIEW

**Comment by maltenuhn** [comment]
So much better.

**Review by Rheeseyb — APPROVED** [review]

File: editor/src/components/canvas/controls/select-mode/border-radius-control.tsx [hunk 1]
```diff
@@ -200,10 +200,13 @@ const CircularHandle = React.memo((props: CircularHandleProp) => {
         <div
           style={{
             visibility: shouldShowHandle ? 'visible' : 'hidden',
-            width: size,
-            height: size,
+            width: 6,
```
Comment (seanparsons) [inline editor/src/components/canvas/controls/select-mode/border-radius-control.tsx:203]: Are these sixes related to the ones in `BorderRadiusHandleSize`?
